components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIAdmin.java

```diff
@@ -175,6 +175,15 @@ void updateMonetizationUsagePublishInfo(MonetizationUsagePublishInfo monetizatio
      */
     List<APICategory> getAllAPICategoriesOfTenant(int tenantID) throws APIManagementException;
 
+    /**
+     * Returns all api categories of the tenant with number of APIs for each category
+     *
+     * @param tenantID
+     * @return
```

Comment:
Shall we add descriptions to the java doc comments?

Reply:
Need to remove @return as it is a void method 
Fixed with d1a24a8